test/integration2/schema/foreign-keys.spec.js

```diff
@@ -86,6 +86,58 @@ describe('Schema', () => {
             }
           });
 
+          it('generates correct SQL for the new foreign key operation with an on delete clause', async () => {
+            const builder = knex.schema.alterTable(
+              'foreign_keys_table_one',
+              (table) => {
+                table
+                  .foreign('fkey_three')
+                  .references('foreign_keys_table_three.id')
+                  .withKeyName('fk_fkey_threeee')
+                  .onDelete('CASCADE');
+              }
+            );
+
+            const queries = await builder.generateDdlCommands();
+
+            if (isSQLite(knex)) {
```

Comment:
If this test only makes sense for SQLite, can you add
`          if (!isSQLite(knex)) {
            return;
          }
`

to it?

Reply:
On it.